presto-main/src/main/java/com/facebook/presto/secretsManager/SecretsManagerHandler.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.secretsManager;
+
+import com.facebook.presto.spi.secretsManager.SecretsManagerFactory;
+import com.google.common.annotations.VisibleForTesting;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static com.facebook.presto.util.PropertiesUtil.loadProperties;
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+public class SecretsManagerHandler
+{
+    private final Map<String, SecretsManagerFactory> secretsManagerFactories = new ConcurrentHashMap<>();
+
+    private static final File SECRETS_MANAGER_CONFIGURATION = new File("etc/secrets-manager.properties");
+
+    private static final String SECRETS_MANAGER_PROPERTY_NAME = "secrets-manager.name";
+
+    private static final Map<String, String> secretsManagerProperties = new ConcurrentHashMap<>();
+
+    /**
+     * Add the secretsmanagerfactory object from plugin to the map
+     * @param secretsManagerFactory
```

Comment:
Here and elswhere, please remove the Javadoc annotations of parameters that have no explanation.  No explanation is needed or desired if the variable name is self-explanatory (as `secretsManagerFactory` is, for example).